extending.md

```diff
@@ -55,17 +56,21 @@ Shadow methods must be marked with the `@Implementation` annotation. Robolectric
 public class ShadowImageView extends ShadowView {
   ...	
   @Implementation
-  public void setImageResource(int resId) {
+  protected void setImageResource(int resId) {
     // implementation here.
   }
 }
 ```
 
+Robolectric supports shadowing all methods on the original class, including private, static, final or native.
```

Comment:
Convention is to wrap code/keywords/symbol names in backticks.

Reply:
Done